lib/utils/utilities.js

```diff
@@ -61,24 +61,23 @@ function getSimulatorsFolder () {
 }
 
 function getSimulatorModelId (cli, target) {
-    let findSimCommand;
-    if (target) {
-        findSimCommand = cli + ' run --list --emulator' + module.exports.PARAMEDIC_COMMON_CLI_ARGS + ' | grep ' + target + ' | tail -n1';
-    } else {
-        findSimCommand = cli + ' run --list --emulator' + module.exports.PARAMEDIC_COMMON_CLI_ARGS + ' | grep ^iPhone | tail -n1';
-    }
+    target = target || '^iPhone';
+
+    // Command to fetch for a list of simulators by target. defaults to (^iPhone)
+    const cmd = `${cli} run --list --emulator${module.exports.PARAMEDIC_COMMON_CLI_ARGS} | grep "${target}" | tail -n1`;
```

Comment:
Is `grep ^iPhone` really equal to `grep "^iPhone"`?

Reply:
Yes, I compared outputs for both commands and got same results.

**Commands:**
```
$ cordova run --list --emulator --no-telemetry --no-update-notifier | grep ^iPhone
$ cordova run --list --emulator --no-telemetry --no-update-notifier | grep "^iPhone"
```

**Results: **
```
iPhone-5s, 12.2
iPhone-6, 12.2
iPhone-6-Plus, 12.2
iPhone-6s, 12.2
iPhone-6s-Plus, 12.2
iPhone-7, 12.2
iPhone-7-Plus, 12.2
iPhone-8, 12.2
iPhone-8-Plus, 12.2
iPhone-SE, 12.2
iPhone-X, 12.2
iPhone-XS, 12.2
iPhone-XS-Max, 12.2
iPhone-XR, 12.2
```

Reply:
macOS+Linux only I guess, but that's probably fine enough here with iOS being macOS only anyway.